src/data/markdown/docs/01 guides/02 Using k6/14 Executors.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Executors
+excerpt: ''
+hideFromSidebar: false
+---
+
+k6 v0.27.0 introduces the concept of _executors_: configurable modes of executing a
+JavaScript function that can model diverse traffic scenarios in load tests.
+
+Multiple executors can be used in the same script, they can be scheduled to run
+parallel or in sequence, and each executor can run a different JS function with
+different environment variables and tags, so this brings a lot of flexibility with
+organizing and modeling testing scenarios.
+
+Existing global execution options such as `vus`, `duration` and `stages` were
+formalized into standalone executors, while new executors were added to support more
+advanced modes of execution. A major benefit of this restructuring is that support
+for new testing scenarios can be added relatively easily to k6, making it very
+extensible.
+
+> ### ⚠  Backwards compatibility
+>
+> Note that pre-v0.27.0 scripts and options should continue to work
+> the same as before (with a few minor breaking changes, as mentioned
+> in the [release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0))
+> but please [create a bug issue](https://github.com/loadimpact/k6/issues/new?labels=bug&template=bug_report.md)
+> if that's not the case.
+
+
+## Executor types
+
+
+### Shared iterations
+
+A fixed number of iterations are "shared" by all VUs, and the test ends once all
+iterations are executed. This executor is equivalent to the global `vus` and
+`iterations` options.
```

Comment:
We should mention the unique identifier name for each executor here (`shared-iterations` in this case), even though it's easy to deduce. Any ideas where? Maybe in parenthesis after the header?

Reply:
Actually, it might be better if there's a small "Examples" section under each executor, so that it's clear what the unique identifier is. Initially I opted against this as these examples will essentially only differ in the `options`, which are already documented, so it seems redundant.

But after more consideration, I'd rather have these small examples there, and rename the current standalone "Examples" to "Advanced examples" where 2 or more executors are used at the same time and to showcase other more complex behavior.

Thoughts?